ModManager/InstallerSystem/Installers/GeneralInstaller.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Modio.Models;
+using ModManager.ModSystem;
+using ModManager.PersistenceSystem;
+
+namespace ModManager.InstallerSystem.Installers
+{
+    public class GeneralInstaller : IModInstaller
+    {
+        private readonly PersistenceService _persistenceService;
+
+        private readonly InstalledModRepository _installedModRepository;
+
+        public GeneralInstaller()
+        {
+            _persistenceService = PersistenceService.Instance;
+            _installedModRepository = InstalledModRepository.Instance;
+        }
+
+        public bool Install(Mod mod, File file)
+        {
+            Manifest manifest = new Manifest(mod, file, "");
+
+            _persistenceService.SaveObject(manifest, Paths.Mods + "/manifest.json");
```

Comment:
I don't really know, but is there a chance the "Paths.Mods + "/manifest.json" " might not work on all OSes? Maybe usisng Path.Combine(Paths.Mods, "manifest.json") is safer?

Reply:
Yes, this was just for test this whole file is yours :) this needs to be the default mod instllation